deepchem/models/tests/test_weavemodel_pytorch.py

```diff
@@ -340,3 +295,77 @@ def test_weave_model_reload():
     original_predict = original_model.predict(dataset)
     reloaded_predict = reloaded_model.predict(dataset)
     assert np.all(original_predict == reloaded_predict)
+
+
+@pytest.mark.torch
+def test_weave_singletask_classification_overfit():
+    """Test weave model overfits tiny data."""
+    # np.random.seed(123)
+    # torch.manual_seed(123)
+    n_tasks = 1
+    current_dir = os.path.dirname(os.path.abspath(__file__))
+
+    # Load mini log-solubility dataset.
+    featurizer = dc.feat.WeaveFeaturizer()
+    tasks = ["outcome"]
+    input_file = os.path.join(current_dir, "assets/example_classification.csv")
+    loader = dc.data.CSVLoader(tasks=tasks,
+                               feature_field="smiles",
+                               featurizer=featurizer)
+    dataset = loader.create_dataset(input_file)
+
+    classification_metric = dc.metrics.Metric(dc.metrics.accuracy_score)
+
+    batch_size = 10
+    model = WeaveModel(n_tasks,
+                       batch_size=batch_size,
+                       learning_rate=0.0003,
+                       dropout=0.0,
+                       mode="classification")
+
+    # Fit trained model
+    model.fit(dataset, nb_epoch=300)
```

Comment:
How long does it take for the test to run for 300 epochs?

Reply:
Around 2 mins